src/Redux/Missions/MissionsSlice.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-param-reassign */
```

Comment:
- Hey, please, kindly remove this eslint disable and try fixing the error as the linters as there to help you write clean and maintainable code as this helps improve our development skills. let's kindly fix the linter.

Reply:
Hi@zieeco, the eslint disable line has been removed and we fixed the error, thank you for your feedback.